scriptmodules/supplementary/moonlight-embedded.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+# @autor: Nyaran
+# @date: 2016/10/09
+# @usage: Place this script into RetroPie-Setup/scriptmodules/supplementary directory and execute the following command:
+#         `sudo RetroPie-Setup/retropie_packages.sh moonlight-embedded`
+#         Also you can use the setup script. the installer is located under *exp packages* option.
+
+
+rp_module_id="moonlight-embedded"
+rp_module_desc="Moonlight Embedded Game Streaming"
+rp_module_section="exp"
+
+function depends_moonlight-embedded() {
+    getDepends libopus0 libexpat1 libasound2 libudev0 libavahi-client3 libcurl3 libevdev2 libenet7 libraspberrypi0 libssl-dev libopus-dev libasound2-dev libudev-dev libavahi-client-dev libcurl4-openssl-dev libevdev-dev libexpat1-dev libpulse-dev uuid-dev libenet-dev

```

Comment:
if the raspberry pi libs is needed - you should include libraspberrypi-dev instead of libraspberrypi0